main.c

```diff
@@ -190,9 +191,9 @@ keypress(XEvent *e)
 void
 scroll_up(unsigned int delta, bool fl_enabled)
 {
-    if ((delta > 0) && fl_enabled) {
+    if (delta > 0 && fl_enabled)
         flashlight.delta_radius += 250.0f;
-    } else {
```

Comment:
Id prefer for nested logic with an else clause to be wrapped in braces for easier readability.